src/components/primitives/Time.tsx

```diff
@@ -15,8 +15,8 @@ export default function Time({ timestamp, prefix }: TimeProps) {
       .replace(/in\s/, prefix ? `${prefix} ` : "In ")
       .replace(/\bminute(s?)\b/g, "m")
       .replace(/\bsecond(s?)\b/g, "s")
-      .replace(/\bhour(s?)\b/g, (_match, plural) => (plural ? "hours" : "hour"))
-      .replace(/\bday(s?)\b/g, (_match, plural) => (plural ? "days" : "day"))
+      .replace(/\bhour(s?)\b/g, "hours")
```

Comment:
Does this works ? If it works nice refacto :p

FROM > .replace(/\bhour(s?)\b/g, (_match, plural) => (plural ? "hours" : "hour"))
TO > .replace(/\bhour(s?)\b/g, "hours")

Reply:
handling plurals